examples/map_transformations/azimuthal_equidistant_projection.py

```diff
@@ -0,0 +1,92 @@
+"""
+===============================================
+Azimuthal Equidistant Projection with Reproject
+===============================================
+
+In this example, we will reproject an AIA image into an
+`azimuthal equidistant projection <https://en.wikipedia.org/wiki/Azimuthal_equidistant_projection>`__.
+You will need v0.6 or higher  of the `reproject` package installed.
```

Comment:
```suggestion
```
I realize that this line is present in other examples, but it's vestigial now that `reproject` is a non-optional dependency.